version.py

```diff
@@ -34,7 +34,9 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 339  # Last bumped for reaction events.
+API_FEATURE_LEVEL = (
+    340  # Last bumped for updating description and permission settings for deactivated groups.
+)
```

Comment:
We should try to be more brief so this never line-wraps; the sole purpose of the text is to cause merge conflicts.

Reply:
``` diff
diff --git a/version.py b/version.py
index 022f29657c..f4a53880ed 100644
--- a/version.py
+++ b/version.py
@@ -34,9 +34,7 @@ DESKTOP_WARNING_VERSION = "5.9.3"
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = (
-    340  # Last bumped for updating description and permission settings for deactivated groups.
-)
+API_FEATURE_LEVEL = 340  # Last bumped for updating settings for deactivated groups.
 
 # Bump the minor PROVISION_VERSION to indicate that folks should provision
 # only when going from an old version of the code to a newer version. Bump
```